ProjectX/ProjectX/MainWindow.xaml.cs

```diff
@@ -29,9 +29,10 @@ namespace ProjectX
     /// </summary>
     public partial class MainWindow : Window
     {
-        
-        private string[] IMAGES = { "projectx_4.jpg", "projectx_7.jpg", "projectx_12.jpg", "projectx_13.jpg", "projectx_15.jpg", "projectx_17.jpg" };//, "projectx_22.jpg", "projectx_32.jpg", "projectx_39.jpg" };//, "projectx_43.jpg", "projectx_44.jpg", "projectx_45.jpg", "projectx_55.jpg", "projectx_61.jpg" };    // images
-        private string[] NAMES = { "alzheimer", "bailey", "bethe", "blackwell", "bolton", "buck" };
+        public string[] IMAGES = new string[98];
```

Comment:
Hard coded the array size. So this will have to be changed whenever a new image is entered.